### PR TITLE
Shim translator rpo set

### PR DESCRIPTION
**Pull request by carlosmerec**
- Add unit test to test shim translation
   - Fix incorrect test test_get_returns_default_for_dual_region_bucket

Depends on merging this [pr](https://github.com/GoogleCloudPlatform/gsutil/pull/1649) first

### CODE REVIEW

File: gslib/tests/test_rpo.py [hunk 1]
```diff
@@ -102,7 +144,7 @@ def _verify_get_returns_default_or_none(self, bucket_uri):
 
   @SkipForXML('RPO only runs on GCS JSON API.')
   def test_get_returns_default_for_dual_region_bucket(self):
-    bucket_uri = self.CreateBucket(location='us')
+    bucket_uri = self.CreateBucket(location='nam4')
```
Comment (dilipped) [inline gslib/tests/test_rpo.py:147]: Curious about this change.

File: gslib/commands/rpo.py [hunk 2]
```diff
@@ -80,6 +81,21 @@
 _set_help_text = CreateHelpText(_SET_SYNOPSIS, _SET_DESCRIPTION)
 _get_help_text = CreateHelpText(_GET_SYNOPSIS, _GET_DESCRIPTION)
 
+GET_COMMAND = GcloudStorageMap(gcloud_command=[
```
Comment (dilipped) [inline gslib/commands/rpo.py:84]: I saw another PR covering this. Do you have chain of PRs. That's gonna be tricky in git.

**Review by dilipped — COMMENTED** [review]
Please run the parity_check.py script and share the output

File: gslib/tests/test_rpo.py (hunk 1 above)
Comment (carlosmerec) [reply gslib/tests/test_rpo.py:147]: Test tries to create dual region bucket but it is passing the zone `us`  which creates a `multi-region` bucket, the test was passing due to only rpo config for `multi-region` bucket is `DEFAULT`, [reference](https://cloud.google.com/storage/docs/json_api/v1/buckets/patch), so when the assertion is made, returns `DEFAULT`, so I changed the code to actually create `dual-region` bucket using a [predefined dual-region](https://cloud.google.com/storage/docs/locations#predefined-dr)

File: gslib/commands/rpo.py (hunk 2 above)
Comment (carlosmerec) [reply gslib/commands/rpo.py:84]: Yes, I do, I'll have to rebase my first branch 😅

File: gslib/tests/test_rpo.py (hunk 1 above)
Comment (dilipped) [reply gslib/tests/test_rpo.py:147]: got it, thanks

**Pull request merged**